spec/ics-3-connection-semantics/closing_equivocation.tex

```diff
@@ -0,0 +1,29 @@
+\include{common}
+
+\begin{document}
+\begin{varwidth}{\linewidth}
+
+\begin{algorithmic}
+\Struct[Datagram]{CONNCLOSEEQUIVOCATION}
+\Field{Identifier}{identifier}
+\Field{Header}{firstHeader}
+\Field{Header}{secondHeader}
+\EndStruct
+\end{algorithmic}
+
+\begin{algorithmic}
+\Function{handleDatagram}{datagram}
+  \Switch{$datagram$}
+    \Case{$CONNCLOSEEQUIVOCATION$}
```

Comment:
Nit (and not important yet): maybe using SNAKE_CASE or CamelCase can help make this a bit more readable? e.g. `$CONN_CLOSE_EQUIVOCATION`

Reply:
Switched to CamelCase.